tests/common/utilities.py

```diff
@@ -668,3 +668,38 @@ def update_environ(*remove, **update):
         env.update(to_restore)
         for k in to_removed:
             env.pop(k)
+
+def get_plt_reboot_ctrl(duthost, tc_name, reboot_type):
+    """
+    @summary: utility function returns list of reboot dict containing timeout and wait
+    for each reboot type
+    @return a list of reboot dict containing timeout and wait for each reboot type
+    DUTHOST:
+        plt_reboot_dict:
+          cold:
+            timeout: 300
+            wait: 600
+          warm-reboot:
+            timeout: 300
+            wait: 600
+          acl/test_acl.py::TestAclWithReboot:
+            timeout: 300
+            wait: 600
+          platform_tests/test_reload_config.py::test_reload_configuration_checks:
+            timeout: 300
+            wait: 60
+    """
+
+    reboot_dict = dict()
+    dut_vars = duthost.host.options['inventory_manager'].get_host(duthost.hostname).vars
```

Comment:
Where is the update to platform/HWSKU inventory file (catering to these various reboot classes timeout, wait time)?

Reply:
Hi @shyam77git, lab/inventory file is testbed specific. An example of what needs to be added in there in the description and in the function get_plt_reboot_ctrl